nominees/nvda.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "NVDA",
-  "description": "NVDA allows blind and vision impaired people to access and interact with the Windows operating system and many third party applications.",
-  "website": "",
+  "aliases": [
+    ""
+  ],
+  "description": "NVDA, the free and open source screen reader for Microsoft Windows. NVDA allows blind and vision impaired people to access and interact with the Windows operating system and many third party applications.",
+  "website": "https://www.nvaccess.org/",
   "license": [
     {
       "spdx": "GPL-2.0",
```

Comment:
There is no license file in the repository. Please have them add the GPL license.